caas/Dockerfile

```diff
@@ -34,4 +37,15 @@ COPY jujuc /opt/
 COPY containeragent /opt/
 COPY pebble /opt/
 
+RUN chown -R juju /var/lib/juju
+RUN chown juju /opt/jujud
+RUN chown juju /opt/jujuc
+RUN chown juju /opt/containeragent
+RUN chown juju /opt/pebble
+RUN chown -R juju /root
```

Comment:
This ideally would still be owned by root and instead have a home directory for juju.

Reply:
Agreed. We should not be changing the permissions /root in any event. Have opt owned by root and sub directories owned by Juju